tox.ini

```diff
@@ -31,4 +31,6 @@ uv_sync_flags = --only-group, lint
 [testenv:run]
 description = Run the development server with auto-reload for code changes.
 package = editable
-commands = uvicorn crawlspace.main:app --reload
+set_env =
```

Comment:
If you wanted to run pytest outside of tox for some reason, I guess you'd just export the env var before doing so?